ark/utils/load_utils.py

```diff
@@ -479,3 +482,115 @@ def load_tiled_img_data(data_dir, fovs, expected_fovs, channel, single_dir, file
+        ome_file_path: pathlib.Path = pathlib.Path(ome_save_dir) / f"{fov_name}.ome.tiff"
+
+        # Set metadata for the OME-TIFF
+        _metadata = {
+            "axes": "CYX",
+            "Channel": {"Name": fov.channels.values.tolist()},
+            "Name": fov_name
+        }
+
+        # Write the OME-TIFF
+        with TiffWriter(ome_file_path, ome=True) as ome_tiff:
+            ome_tiff.write(
+                data=fov.values,
+                photometric="minisblack",
+                compression=_compression["algorithm"],
+                compressionargs=_compression["args"],
+                metadata=_metadata
+            )
+
+
+def ome_to_fov(ome: Union[str, pathlib.Path], data_dir: Union[str, pathlib.Path]) -> None:
+    """
+    Converts an OME-TIFF with n channels to a FOV (A folder consisting of those n channels).
+    The folder's name is given by the Image `@Name` in the xml metadata.
+
+    Args:
+        ome (Union[str, pathlib.Path]): The path to the OME-TIFF file.
+        data_dir (Union[str, pathlib.Path]): The path where the FOV will be saved.
+    """
+    iou.validate_paths([ome, data_dir], data_prefix=False)
+
+    with TiffFile(ome, is_ome=True) as ome_tiff:
+        # String representation of the OME-XML metadata & convert to dictionary
+        ome_xml_metadata: OrderedDict = xmltodict.parse(ome_tiff.ome_metadata.encode())
+
+        # An OME-TIFF's OME-XML metadata has either a single Image (dict), or a list of Images
+        # (List[dict]). IF it's a list of images, then grab the first image (all images should
+        # be the same, just different resolutions)
+        if isinstance(ome_xml_metadata["OME"]["Image"], dict):
+            image_data = ome_xml_metadata["OME"]["Image"]
+        else:
+            image_data = ome_xml_metadata["OME"]["Image"][0]
+
+        # Get the OME-XML image name
+        image_name: str = ome.stem.split(".")[-2]
+
+        # Get the OME-XML channel metadata
+        channel_metadata: List[OrderedDict] = image_data["Pixels"]["Channel"]
+        save_dir: pathlib.Path = pathlib.Path(data_dir) / image_name
+
+        # Corner case when only one channel. (OME-XML is a dict instead of a list of dicts)
+        if isinstance(channel_metadata, dict):
+            channel_metadata = [channel_metadata]
+
+        # Get the channel names. Ex: {"DAPI", "CD3", "CD8"}.
+        # No need to check for ordering, as the OME-TIFF Channel data is ordered.
+        if "@Name" in channel_metadata[0].keys():
+            channels: List[str] = [c["@Name"] for c in channel_metadata]
+        else:
+            channels: List[str] = [f"Channel {c}" for c in range(len(channel_metadata))]
+
+        # Get the TIFF pages for the current image
+        ome_tiff_img_pages: TiffPageSeries = ome_tiff.series[0].pages
+
+        for ome_tiff_page, channel in zip(ome_tiff_img_pages, channels):
+            save_dir.mkdir(parents=True, exist_ok=True)
+
+            image_utils.save_image(
```

Comment:
I think it would be better to hold off changing more of the imports than we need to for now. We should put out a new python-only release first, make sure everything works, and then we can make additional changes on top of that.

Reply:
@ngreenwald Sounds good, I'll get a new release in.